package.json

```diff
@@ -114,7 +116,7 @@
     "bash": "0.0.1",
     "bluebird": "^3.3.3",
     "body-parser": "^1.14.1",
-    "capitano": "^1.8.2",
+    "capitano": "^1.9.0-60-preserve-raw-string-values-7051fdce11705b5506434084b99821f955715a18",
```

Comment:
This needs to be updated after the Capitano PR is merged. Meanwhile, it should allow the CI tests to pass.